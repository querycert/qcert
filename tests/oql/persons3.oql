/* SFW with distinct */
select distinct d
from c in Marketers,
     d in c->clients
