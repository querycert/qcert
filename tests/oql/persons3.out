["123", "124", "125", "102", "100", "101"]